package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-htmlbars": "^4.3.1",
     "ember-cli-postcss": "^6.0.1",
     "ember-composable-helpers": "^4.1.1",
+    "ember-responsive-image": "^2.0.0",
```

Comment:
This should probably be v3, we don't want to have to do this again.

https://github.com/kaliber5/ember-responsive-image/releases